src/config/production.ts

```diff
@@ -20,6 +20,9 @@ const config: ConfigInterface = {
     domain: 'https://auth.iasql.com/',
     audience: 'https://api.iasql.com', // id of this api in auth0
   },
+  cors: {
+    origin: 'https://api-staging.iasql.com'
```

Comment:
This one should just be:

```suggestion
    origin: 'https://api.iasql.com'
```